### ISSUE TITLE
SEO not working

### ISSUE THREAD

**Issue by jonnysmillie** [LekoArts/gatsby-starter-portfolio-emilia#6]
Hi again, I just thought I'd make you aware that the SEO part of this starter doesn't seem to work correctly. Currently it seems that basic SEO is not set such as title and description. I haven't found the source of the problem yet but none of the basic stuff seems to get set.

**Comment by jonnysmillie** [comment]
**More info**

I noticed this when I saw Google displaying "This app works best with JavaScript enabled." as the site title when the site is listed in search results. 

I then used a few common SEO crawlers on both my site and the demo site of this project and they showed that neither sites had title or description set (amongst other things).

It seems strange because when inspecting the elements the meta tags are present and with the correct values but they do not seem to get picked up by the crawlers.

The site has no errors and is functioning totally correctly in every other regard.
